x-pack/legacy/plugins/fleet/public/lib/adapters/agent/rest_agent_adapter.ts

```diff
@@ -20,12 +21,32 @@ export class RestAgentAdapter extends AgentAdapter {
 
   public async get(id: string): Promise<Agent | null> {
     try {
-      return (await this.REST.get<ReturnTypeGet<Agent>>(`/api/fleet/agent/${id}`)).item;
+      return (await this.REST.get<ReturnTypeGet<Agent>>(`/api/fleet/agents/${id}`)).item;
     } catch (e) {
       return null;
     }
   }
 
+  public async getAgentEvents(
+    id: string,
+    search: string,
+    page: number,
+    perPage: number
+  ): Promise<{
```

Comment:
for agent list i updated the return type to [`ReturnTypeList`](https://github.com/elastic/kibana/pull/47621/files#diff-84ce891471901a1926e0e201d08f3425R35) (which I added [`perPage`](https://github.com/elastic/kibana/pull/47621/files#diff-3e627a4b8b50cef2bce16439e9139389R85) property to), what do you think of using the same?

Reply:
make sense to use it here too